src/test/kotlin/com/charleskorn/kaml/YamlWritingTest.kt

```diff
@@ -362,7 +478,8 @@ object YamlWritingTest : Spek({
                     )
                 )
 
-                val serializer = MapSerializer(String.serializer(), MapSerializer(String.serializer(), String.serializer()))
+                val serializer =
```

Comment:
Could you please revert all of the unrelated formatting changes here and below? (this helps keep the diff clean and focused on the changes you've made)